app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -414,6 +414,10 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+
```

Comment:
@bhaktideshmukh Please remove this extra line same goes for other dimen.xml

Reply:
Done!!